src/objectid.ts

```diff
@@ -194,7 +194,7 @@ export class ObjectId extends BSONValue {
    *
    * @param otherId - ObjectId instance to compare against.
    */
-  equals(otherId: string | ObjectId | ObjectIdLike): boolean {
+  equals(otherId: string | ObjectId | ObjectIdLike | undefined | null): boolean {
     if (otherId === undefined || otherId === null) {
```

Comment:
I think we could maybe simplify this now to:

```suggestion
    if (otherId == null) {
```

Reply:
This will only change the runtime implementation of the code. The types change of the parameter in this function still should be updated to accept `null` and `undefined` values.

Reply:
Yes, I was just thinking of also simplifying the runtime implementation as well along with the type change while we are here. But it's just a nit, I'm happy to merge if no one feels strongly about it.